docs/update_and_migration/migrate_to_ibexa_dxp/migrating_from_ez_publish_platform.md

```diff
@@ -242,6 +242,69 @@ Import to your database the changes provided in one of the following files. It's
 
 Steps here should only be done once you are ready to move away from legacy and Legacy Bridge, as the following Field Types are not supported by legacy. In other words, content you have migrated will not be editable in legacy admin interface anymore, but rather in the more modern eZ Platform back-end UI, allowing you to take full advantage of what the Platform has to offer.
 
+The Field Types unsupported in eZ Platform are:
+
+- [XmlText](#321-migrate-xmltext-content-to-richtext)
+- [Page](#322-migrate-page-field-to-page-ez-enterprise-only)
+- Star Rating
+
+For Field Types which do not have specific procedures below, you must either:
```

Comment:
If you go with:
```suggestion
For Field Types which do not have specific procedures below, you must take one of the following courses of action:
```
you can remove the ", or" part and fullstop from the bulleted list below.